andela/Titan-Devs
# #163518701 Bookmark an article to read it later

#### What does this PR do?
- It helps the logged in user to bookmark an article for reading it later
#### Description of Task to be completed?
- Make sure that the user will be able to bookmark an article
- If that article is bookmarked by him, the bookmark will be removed/deleted

#### How should this be manually tested?
- After cloning the repository, check out `ft-create-article-endpoints-163518689` branch
- Make sure to read `README.md` file to get the setup instructions
- Using any `API client tool(usually POSTMAN)`,
- Create the user and login to get the `access-token`
- Create an article using `POST /api/v1/articles` endpoint
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
- The request's body should be as mentioned in the `README.md` file under `Endpoints:` ==> `Add Comments to an Article` subtitle
After the successful article creation,
- Bookmark that article using this endpoint ==> `POST /api/v1/articles/:slug/bookmark`. It will only require passing the {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR in the request headers and passing the correct article's slug as mentioned above in request URL.

#### Any background context you want to provide?
I deleted `helpers/data.js` as it contained the testData that we already have in `tests/helpers/testData`
#### What are the relevant pivotal tracker stories?
- #163518701
